tests/Auth0Test.php

```diff
@@ -313,7 +294,7 @@ public function testThatRenewTokensSucceeds()
 
         $mock = new MockHandler( [
             // Code exchange response.
-            new Response( 200, self::$headers, '{"access_token":"1.2.3","refresh_token":"2.3.4"}' ),
+            new Response( 200, self::$headers, '{"access_token":"1.2.3","refresh_token":"2.3.4","id_token":"'.$id_token.'"}' ),
```

Comment:
The refresh token response would also omit an ID token if the original code exchange did not include one. Leaving this out makes this test inaccurate for how the process would work in a real application. With this ID token added, the nonce is checked and removed from storage before the tokens can be renewed. This then causes the exception that was reported in #432